app/login/page.tsx

```diff
@@ -3,24 +3,22 @@ import { useState, ChangeEvent, useEffect } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/navigation';
 import Logo from '@/components/Logo/Logo';
-import logo from '/public/assets/logo-colored-outline.svg';
+import logo from '@/public/assets/logo-colored-outline.svg';
 import { Input } from '@/components/Input/Input';
 import { Button } from '@/components/Button/Button';
-import { loginAccount } from '@/api/apiFunctions';
+import { useAuthContext } from '@/context/AuthContextProvider';
 
-export default function Login({
-  searchParams,
-}: {
-  searchParams: { message: string };
-}) {
+export default function Login() {
   const router = useRouter();
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
-  let session: any;
+  const { loginAccount, isSignedIn } = useAuthContext();
 
-  if (typeof window !== 'undefined') {
```

Comment:
I missed this during the impromptu meeting @ Discord 5/7/2024. I know local storage was a temporary solution for storing cookies, but have we confirmed that isSignedIn works as intended here with router.push()?

Reply:
I have it all working with router.push.  I created a new isSignedIn in the context file.